tests/integration/operatorhub/cmd_link_test.go

```diff
@@ -131,7 +131,7 @@ var _ = Describe("odo link command tests for OperatorHub", func() {
 					helper.Cmd("odo", "link", svcFullName, "--bind-as-files", "--name", bindingName, "--context", commonVar.Context).ShouldPass()
 				})
 
-				It("should display the link between the components in odo describe", func() {
+				It("should display component links in odo describe", func() {
```

Comment:
Did you change this to avoid the duplication report from Sonar Cloud?

Reply:
Yes

Reply:
Probably don't do that. Rebasing on top of the main branch will mostly make the Sonar cloud duplication error go away.